packages/bff/src/plugins/verifyToken.ts

```diff
@@ -0,0 +1,87 @@
+import axios from 'axios';
+import { FastifyPluginAsync, FastifyReply, FastifyRequest, HookHandlerDoneFunction } from 'fastify';
+import fp from 'fastify-plugin';
+import jwt from 'jsonwebtoken';
+import config from '../config';
+import { IdTokenPayload, SessionStorageToken } from './oidc';
+export async function getIsTokenValid(request: FastifyRequest): Promise<boolean> {
```

Comment:
I think it makes more sense to have this as the same plugin as `oidc.ts`, they go hand in hand. If you want to have it in two files then maybe we recreate an `oidc/` folder and keep `oidc/index.ts` as well as `oidc/verifyToken.ts`.

Imagine that we wanted to publish this to npm, it doesn't make that much sense that the end user would have to activate two different plugins.

Reply:
I strongly disagree ---- business logic for validating a token has nothing to do with oidc